src/test/ui/specialization/specialization-default-methods.rs

```diff
@@ -55,8 +55,9 @@ trait Bar {
 //                   /  \
 //            Vec<i32>  $Vec<i64>
 
-// use the provided method
-impl<T> Bar for T {}
+impl<T> Bar for T {
+    default fn bar(&self) -> i32 { 0 }
```

Comment:
It would be good to test the negative case where we don't forward `default fn / type / const` and we get errors.